atlasaction/cloud/client.go

```diff
@@ -34,21 +35,28 @@ func (r *roundTripper) RoundTrip(req *http.Request) (*http.Response, error) {
 	return http.DefaultTransport.RoundTrip(req)
 }
 
-// New creates a new Client for the Atlas Cloud API.
-func New(token, version, cliVersion string) *Client {
+func newClient(endpoint, token, version, cliVersion string) *Client {
```

Comment:
why the extra function is needed?

Reply:
i dont want to change the signature of the exported function, since I dont understand this code of registrating it:
https://github.com/ariga/atlas-action/blob/67c0c2514b4fb7bcdda8d8c309be777c3df2788a/atlasaction/action.go#L240-L244